content/en/docs/Configuration/p8s-jaeger-grafana/prometheus.md

```diff
@@ -0,0 +1,183 @@
+---
+title: "Prometheus"
+description: >
+  This page describes how to configure Prometheus for Kiali.
+---
+
+
+## Prometheus configuration
+
+Kiali *requires* Prometheus to generate the
+[topology graph]({{< relref "../../Features/topology" >}}),
+[show metrics]({{< relref "../../Features/details#metrics" >}}),
+[calculate health]({{< relref "../../Features/health" >}}) and
+for several other features. If Prometheus is missing or Kiali
+can't reach it, Kiali won't work properly.
+
+By default, Kiali assumes that Prometheus is available at the URL of the form
+`http://prometheus.<istio_namespace_name>:9090`, which is the usual case if you
+are using [the Prometheus Istio
+add-on](https://istio.io/latest/docs/ops/integrations/prometheus/#option-1-quick-start).
+If your Prometheus instance has a different service name or is installed to a
```

Comment:
I think the proper language is if you installed things IN a namespace... you didn't install TO a namespace.

```suggestion
If your Prometheus instance has a different service name or is installed in a
```